mongodbatlas/version.go

```diff
@@ -0,0 +1,15 @@
+package mongodbatlas
```

Comment:
[non-blocking] I don't feel moving this to a version file adds any value vs the mongodbatlas. If we were to have a package `version` then moving _only_ the `ClientVersion` there would make more sense

Reply:
yeah fair enough, updated